src/simulator/human_object.cpp

```diff
@@ -91,6 +94,8 @@ HumanObject::HumanObject(const vector<string>& config_file) :
   reach_goal_threshold_ = CONFIG_reach_goal_threshold;
   mode_ = static_cast<HumanMode>(CONFIG_mode);
 
+  state_publisher_ = n->advertise<HumanStateMsg>(CONFIG_state_topic, 1);
```

Comment:
This is not handling multiple human case, and it will cause problem if you load config of multiple human. My suggestion is to make the human message an array, and put all human message into that array. If you want to visualize the message in rviz, you can just create a markerarray message to visualize them in rviz. Since we want this message type to be used for real tracking system as well, it is better to design this human array messgae well.